activerecord/lib/active_record/database_configurations/database_config.rb

```diff
@@ -13,6 +13,11 @@ def initialize(env_name, spec_name)
         @spec_name = spec_name
       end
 
+      def config
+        ActiveSupport::Deprecation.warn("DatabaseConfig#config will be removed in favor of DatabaseConfigurations#configuration_hash which returns a hash with symbol keys")
```

Comment:
We should update this to say what version it will be removed in which I guess since this is going into 6.1.0 it should be removed in 6.2.0

Reply:
All set!